website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md

```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
 
 To sum things up, the world has changed in many ways! Therefore, storage has changed too, right? Well, not really. Storage is hard. It is far easier through marketing and open source influence peddling to get your 20-year-old project labelled “cloud native” than it is to actually start an entirely new storage system that runs anywhere.
-![https://cdn-images-1.medium.com/max/800/0*vNlv_UcOVV5Y456W](https://lh5.googleusercontent.com/I45Q124TO2nST7bzWOk11g3IWsNc59HdZkrFxpTD5480D6_2w5Px_hdVYn3eVvLzYg3QpD9JSEIF7_EjsG7ckBrhvnm52ETIcytvlLSnG28Ru8I9hIpEiS2ehd_uIIaMWb5DptFe)
+
+![https://cdn-images-1.medium.com/max/800/0*vNlv_UcOVV5Y456W](https://cdn-images-1.medium.com/max/800/0*vNlv_UcOVV5Y456W)
+
 To begin at the bottom of the stack, you not only have to accommodate the presence of extremely fast storage hardware, you also have to accommodate running *on clouds* themselves. How can you do this? And how do you do so in a way that does not itself introduce another single point of failure?
 
 The approach we take at OpenEBS is to run our storage capabilities in a decoupled way so that the controller — that serves the actual workloads — is itself stateless and easily rescheduled. Also, it runs in the user space in containers so that it really can run anywhere. It then coordinates reads and writes to logically underlying data containers.
 
 We also keep the data local and utilize intelligence in our control plane to leverage Kubernetes to optimize data placement of primary data stores and replicas. There is much more I could discuss here, but I’ll leave it at this high level for now.
-![https://cdn-images-1.medium.com/max/800/0*ZFFma1SdaG9GaiAS](https://lh6.googleusercontent.com/RHiDJdReInvNww3Xa8BLpafkAU_Wig2V7zbP45feMmRoqr_nmRFA0W3WIb596vQsSJqjYE3CfjzZYe0b4Qo3jFpKcjDIi3ywXqxPzRmjEFMepS43Z8B-ECoQuWMcoKBL3bzF4a_3)
+
+![https://cdn-images-1.medium.com/max/800/0*ZFFma1SdaG9GaiAS](https://cdn-images-1.medium.com/max/800/0*ZFFma1SdaG9GaiAS)
+
 Secondly, instead of having a one size fits all storage system that is limited in its ability to make trade-offs for each of the potentially thousands of workloads, each workload has its own storage system with OpenEBS. The specific needs of those workloads and of the “two pizza” teams that build and run the workloads are each served by their own OpenEBS. This limits the need for cross-functional team meetings in which members argue about how they want the storage to behave. They simply invoke their particular flavor via a Helm chart and, with the help of the right storage classes, away they go!
-![https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG](https://lh5.googleusercontent.com/WTaj_EcsD5vq_q2EmSxpOByODAp4uOFmX0x8mvCulxvQsG8pGqmpxC0QFk2xnMpCz1qRkkwrP8cAN-7e2PRMmT1cpDFtfUnk2WCptEAbD9xqURIu19rqTxt6Tj4WhVTC-GWrsgoI)
+
+![https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG](https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG)
```

Comment:
There might be some places where the relative path of the image is not correct and that comes as a broken image. Let's look out for those images and then we fix them.